tests/nvmeof-mp/rc

```diff
@@ -35,13 +35,16 @@ and multipathing has been enabled in the nvme_core kernel module"
 		ib_umad
 		nvme-rdma
 		nvmet-rdma
-		rdma_rxe
 		scsi_dh_alua
 		scsi_dh_emc
 		scsi_dh_rdac
 	)
 	_have_modules "${required_modules[@]}" || return
 
+	if [ -z "$use_hw_rdma" ]; then
+		_have_modules rdma_rxe || _have_modules siw
```

Comment:
Does this hunk adds missing siw module check for nvmeof-mp group? If so, I suggest to note it in the commit message.

Reply:
Or even better, add a new patch that implements this change only.